app/models/invenio_rdm_record_converter.rb

```diff
@@ -16,10 +16,34 @@ def initialize(generic_file=nil)
     return unless generic_file
     @@header_lookup ||= HeaderLookup.new
 
-    #PIDs
-    @pids = invennio_pids(generic_file.doi.shift)
-    @metadata = invenio_metadata(generic_file)
-    @provenance = invenio_provenance(generic_file.proxy_depositor, generic_file.on_behalf_of)
+    @record = record_for_export(generic_file)
+    @file = filename_and_content_path(generic_file)
+  end
```

Comment:
This was so the JSON appears with the attributes/metadata and the file nested separately.

Reply:
I like it.